takeshape/penny
# Remove TakeShape profile

### Test Plan (Steps to test):

1. Go to the preview site
1. Sign in, check your account pages, any issues?

This is blocked by https://github.com/takeshape/takeshape-deluxe-sample-project/pull/32 since the product components here aren't safe for all the new fixtures.
